bgpd/bgp_route.c

```diff
@@ -2219,6 +2219,14 @@ static void bgp_process_main_one(struct bgp *bgp, struct bgp_node *rn,
 		}
 	}
 
+	/* advertise/withdraw type-5 routes */
```

Comment:
what's the purpose of this logic? in my test this causes frr to send type-5 routes received back to it's peer